### DISCUSSION TITLE
React 18 Support

### DISCUSSION THREAD

**Discussion by 3lijah** [tinacms/tinacms#2689] [General]
Hello!

I recently had a go at implementing NextJS's React Server Components but I didn't get very far. I wasn't able to install the React 18 package because next-tinacms-cloudinary doesn't seem to support it. I didn't try very hard to go much further.

While I wasn't trying to use React Server Components for TinaCMS content, I'm still interested to know if any work has been done towards the next version of React 18 and the new NextJS features?

**Comment by lukei8ii** [comment]
This is also hamstringing us as well not being able to upgrade to React 18 due to Tina CMS.

**Reply by jamespohalloran** [reply]
Can you explain what you're seeing when upgrading to React18? It should now work.
